src/xSTIR/CMakeLists.txt

```diff
@@ -28,6 +28,9 @@ if (STIR_FOUND)
   ADD_SUBDIRECTORY(pSTIR)
   ADD_SUBDIRECTORY(mSTIR)
 
+  # Going to need this elsewhere
```

Comment:
arguably, this means we should find STIR one level up (in src/CMakeLists.txt). This needs a tiny bit of reorg of the CMakeLists.txt. I guess raise an issue and do it later

Reply:
I think it's ok here, as it keeps the engines separated. The only code that should need it elsewhere is the synergistic stuff. 

And arguably, the synergistic CMake code should be more intellegent (ie. only build against engines that can be found).